### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,2 @@
 # Default owners
-* @rickeyre
-* @alicoding
+* @rickeyre * @alicoding
```
Comment (randysingh) [inline CODEOWNERS]: I think it needs to be * @rickeyre @alicoding
based on https://help.github.com/articles/about-codeowners/

Comment (RickEyre) [inline CODEOWNERS]: I don't think we need the extra * in front of your name.

**Comment by alicoding** [comment]
@RickEyre @randysingh done :)

**Comment by alicoding** [comment]
yay

**Comment by RickEyre** [comment]
Can you rebase your three commits into one? After that it's good to go!

**Comment by alicoding** [comment]
@rickeyre just approve first ;)

**Comment by RickEyre** [comment]
@alicoding It's going to become unapproved once you push new updates 😄

**Pull request merged**